pptx/oxml/shapes/autoshape.py

```diff
@@ -238,8 +239,25 @@ class CT_PresetGeometry2D(BaseOxmlElement):
     """
 
     avLst = ZeroOrOne("a:avLst")
-    prst = RequiredAttribute("prst", MSO_AUTO_SHAPE_TYPE)
+    preset = RequiredAttribute("prst", ST_ShapeType)
 
+    @property
```

Comment:
This was the workaround I mentioned in the description.  The way Steve originally set this up, he is using AutoShapeTypes as the list of possibilities for `prst` and throughout the code, it is expecting a member of that ENUM.  Unfortunately, Connectors are not part of that list.  Both Connectors and AutoShapes inherit from BaseShape but aren't the same.  As a result, I found this little workaround that seems to let us use AutoShapes as usual but also use connectors when we want.